tsconfig.json

```diff
@@ -4,11 +4,14 @@
     "module": "commonjs",
     "declaration": true,
     "outDir": "./lib",
-    "strict": true
+    "strict": true,
+    "resolveJsonModule": true,
+    "esModuleInterop": true,
```

Comment:
check if this is needed?

Reply:
Sir, with flag `esModuleInterop` we can import CommonJS modules in compliance with es6 modules spec.